scripts/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION="$(tr -d '\n' < .tsqllint-version)"
+
+# Remove possible old source
+rm -rf tsqllint/
+
+curl -Ls "https://github.com/tsqllint/tsqllint/archive/v$VERSION.tar.gz" -o tsqllint.tar.gz
```

Comment:
download the file keeping the extension and cache it in CI (curl has options to not download if already present)

Reply:
I guess `curl` doesn't have that option. I'll go with `wget` then.

Reply:
curl does have it as well IIRC